assets/texts/sv/Changelog/1.1.md

```diff
@@ -0,0 +1,9 @@
+* **Featured:** Save QR code size manually or automatically.
```

Comment:
Note you don't need to copy non-translated files, you can just ignore/delete them. But it's fine, if you want remove them though… :slightly_smiling_face:

Reply:
Ok, I went ahead and just translated them all, i figured I might aswell do that.